### DISCUSSION TITLE
Long code wrapping issue of importing `@rspress/plugin-shiki` plugin

### DISCUSSION THREAD

**Discussion by momo91377** [web-infra-dev/rspress#1134] [Q&A] [answered]
目前使用的Rspress版本1.17.1，引入@rspress/plugin-shiki进行代码块高亮后导致了两个问题，第一是代码块的换行按钮无法生效 ，第二是配置的defaultWrapCode: true进行长代码默认自动换行不生效，尝试更新了Rspress 1.22.0版本，此问题依然存在

**Comment by Timeless0911** [comment, marked as answer]
`@rspress/plugin-shiki` doesn't regoconize `codeWrap` related, see https://github.com/web-infra-dev/rspress/blob/7dc36edc7ed9a677cf9f0efd107f4bb75b3fd042/packages/theme-default/src/layout/DocLayout/docComponents/code/index.tsx#L42-L56

Welcome make a PR to support it.

**Reply by momo91377** [reply]
Thanks a lot
